diff --git a/assets/styles/admin.scss b/assets/styles/admin.scss
index e69de29..6d0c464 100644
--- a/assets/styles/admin.scss
+++ b/assets/styles/admin.scss
@@ -0,0 +1 @@
+:root { --sidebar-bg: #deebff; }
diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index b9c1f9b..6159395 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -64,6 +64,7 @@ class DashboardController extends AbstractDashboardController
 
     public function configureAssets(): Assets
     {
-        return parent::configureAssets();
+        return parent::configureAssets()
+            ->addWebpackEncoreEntry('admin');
     }
 }
