diff --git a/assets/styles/admin.css b/assets/styles/admin.css
new file mode 100644
index 00000000..4f8289f6
--- /dev/null
+++ b/assets/styles/admin.css
@@ -0,0 +1,3 @@
+body {
+    background: lightcyan;
+}
diff --git a/webpack.config.js b/webpack.config.js
index 056b04aa..19f27c35 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -21,6 +21,7 @@ Encore
      * and one CSS file (e.g. app.css) if your JavaScript imports CSS.
      */
     .addEntry('app', './assets/app.js')
+    .addStyleEntry('admin', './assets/styles/admin.css')
 
     // enables the Symfony UX Stimulus bridge (used in assets/bootstrap.js)
     .enableStimulusBridge('./assets/controllers.json')
