diff --git a/assets/styles/admin.scss b/assets/styles/admin.scss
new file mode 100644
index 0000000..e69de29
diff --git a/webpack.config.js b/webpack.config.js
index 056b04a..49e0929 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -21,6 +21,7 @@ Encore
      * and one CSS file (e.g. app.css) if your JavaScript imports CSS.
      */
     .addEntry('app', './assets/app.js')
+    .addStyleEntry('admin', './assets/styles/admin.scss')
 
     // enables the Symfony UX Stimulus bridge (used in assets/bootstrap.js)
     .enableStimulusBridge('./assets/controllers.json')
