diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 980d4c01..a883dccb 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -41,6 +41,7 @@ class QuestionCrudController extends AbstractCrudController
                     ->setParameter('enabled', true);
             });
         yield AssociationField::new('answers')
+            ->autocomplete()
             ->setFormTypeOption('choice_label', 'id')
             ->setFormTypeOption('by_reference', false);
         yield Field::new('createdAt')
