diff --git a/src/Entity/User.php b/src/Entity/User.php
index 7dfa27ed..05b12e44 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -81,6 +81,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
         $this->answers = new ArrayCollection();
     }
 
+    public function __toString(): string
+    {
+        return $this->getFullName();
+    }
+
     public function getId(): ?int
     {
         return $this->id;
