diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index a883dcc..47ac7a3 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -42,7 +42,6 @@ class QuestionCrudController extends AbstractCrudController
             });
         yield AssociationField::new('answers')
             ->autocomplete()
-            ->setFormTypeOption('choice_label', 'id')
             ->setFormTypeOption('by_reference', false);
         yield Field::new('createdAt')
             ->hideOnForm();
