diff --git a/src/Entity/User.php b/src/Entity/User.php
index b315232..719e0f0 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -82,6 +82,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
         $this->answers = new ArrayCollection();
     }
 
+    public function __toString(): string
+    {
+        return $this->getFullName();
+    }
+
     public function getId(): ?int
     {
         return $this->id;
