diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 01fd3625..4a6a32db 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -27,7 +27,18 @@ class QuestionCrudController extends AbstractCrudController
         yield Field::new('votes', 'Total Votes')
             ->setTextAlign('right');
         yield AssociationField::new('askedBy')
-            ->autocomplete();
+            ->autocomplete()
+            ->formatValue(function ($value, ?Question $question): ?string {
+                if (!$question) {
+                    return null;
+                }
+
+                if (!$user = $question->getAskedBy()) {
+                    return null;
+                }
+
+                return sprintf('%s&nbsp;(%s)', $user->getEmail(), $user->getQuestions()->count());
+            });
         yield Field::new('createdAt')
             ->hideOnForm();
     }
