diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index a65f6163..ece3790e 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -39,7 +39,7 @@ class Question
     #[ORM\Column]
     private int $votes = 0;
 
-    #[ORM\OneToMany('question', Answer::class)]
+    #[ORM\OneToMany('question', Answer::class, orphanRemoval: true)]
     private Collection $answers;
 
     #[ORM\ManyToOne(inversedBy: 'questions')]
