diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 87dac44..9f77de7 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -23,7 +23,8 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
-        yield ImageField::new('avatar');
+        yield ImageField::new('avatar')
+            ->setBasePath('uploads/avatars');
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
