diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 7afb90d..aa6951e 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,6 +25,7 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
+        yield AvatarField::new('avatar');
         yield ImageField::new('avatar')
             ->setUploadDir('public/uploads')
             ->setBasePath('uploads')
