diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index f0bc4540..9bd7d20a 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,8 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
-        yield ImageField::new('avatar');
+        yield ImageField::new('avatar')
+            ->setBasePath('uploads/avatars');
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
