diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index d29834c..aaf909e 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,8 @@ class UserCrudController extends AbstractCrudController
             ->onlyOnIndex();
         yield ImageField::new('avatar')
             ->setBasePath('uploads/avatars')
-            ->setUploadDir('public/avatars/uploads');
+            ->setUploadDir('public/avatars/uploads')
+            ->setUploadedFileNamePattern('[slug]-[timestamp].[extension]');
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
