diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 7fb3cf1..4c4c38f 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -26,7 +26,8 @@ class UserCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->onlyOnIndex();
         yield ImageField::new('avatar')
-            ->setUploadDir('public/uploads');
+            ->setUploadDir('public/uploads')
+            ->setBasePath('uploads');
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
