diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index b0e3a4f..983cfdf 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -31,7 +31,8 @@ class UserCrudController extends AbstractCrudController
         yield ImageField::new('avatar')
             ->setBasePath('uploads/avatars')
             ->setUploadDir('public/avatars/uploads')
-            ->setUploadedFileNamePattern('[slug]-[timestamp].[extension]');
+            ->setUploadedFileNamePattern('[slug]-[timestamp].[extension]')
+            ->onlyOnForms();
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
