diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 03501c35..ddd2f126 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -28,7 +28,8 @@ class UserCrudController extends AbstractCrudController
         yield AvatarField::new('avatar')
             ->formatValue(static function ($value, User $user) {
                 return $user->getAvatarUrl();
-            });
+            })
+            ->hideOnForm();
         yield ImageField::new('avatar')
             ->setBasePath('uploads/avatars')
             ->setUploadDir('public/uploads/avatars')
