diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 9f77de7..d29834c 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,8 @@ class UserCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->onlyOnIndex();
         yield ImageField::new('avatar')
-            ->setBasePath('uploads/avatars');
+            ->setBasePath('uploads/avatars')
+            ->setUploadDir('public/avatars/uploads');
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
