diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 03a25108..bb3b3f20 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,10 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
-        yield AvatarField::new('avatar');
+        yield AvatarField::new('avatar')
+            ->formatValue(static function ($value, User $user) {
+                return $user->getAvatarUrl();
+            });
         yield ImageField::new('avatar')
             ->setBasePath('uploads/avatars')
             ->setUploadDir('public/avatars/uploads')
