diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index ccee8fe2..6840bb7f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -41,7 +41,8 @@ class QuestionCrudController extends AbstractCrudController
                     'disabled' => $pageName !== Action::NEW,
                 ],
             ]);
-        yield Field::new('name');
+        yield Field::new('name')
+            ->setSortable(false);
         yield TextareaField::new('question')
             ->hideOnIndex();
         yield VotesField::new('votes', 'Total Votes')
