diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 34833dd..e09735d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -40,7 +40,8 @@ class QuestionCrudController extends AbstractCrudController
                     'disabled' => $pageName !== Action::NEW,
                 ],
             ]);
-        yield Field::new('name');
+        yield Field::new('name')
+            ->setSortable(false);
         yield TextareaField::new('question')
             ->hideOnIndex();
         yield Field::new('votes', 'Total Votes')
