diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 4796141..5eea58d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -39,7 +39,8 @@ class QuestionCrudController extends AbstractCrudController
                 'disabled',
                 $pageName !== Crud::PAGE_NEW
             );
-        yield Field::new('name');
+        yield Field::new('name')
+            ->setSortable(false);
         yield TextareaField::new('question')
             ->hideOnIndex();
         yield VotesField::new('votes', 'Total Votes')
