diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 175c685..734b064 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -19,6 +19,15 @@ class QuestionCrudController extends AbstractCrudController
         return Question::class;
     }
 
+    public function configureCrud(Crud $crud): Crud
+    {
+        return parent::configureCrud($crud)
+            ->setDefaultSort([
+                'askedBy.enabled' => 'DESC',
+                'createdAt' => 'DESC',
+            ]);
+    }
+
     public function configureFields(string $pageName): iterable
     {
         yield IdField::new('id')
