diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index c492fbc..19dbe75 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -49,7 +49,8 @@ class QuestionCrudController extends AbstractCrudController
                 return [
                     'slug' => $question->getSlug(),
                 ];
-            });
+            })
+            ->addCssClass('btn btn-success');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
