diff --git a/src/EasyAdmin/VotesField.php b/src/EasyAdmin/VotesField.php
index a480451..8f9a6f0 100644
--- a/src/EasyAdmin/VotesField.php
+++ b/src/EasyAdmin/VotesField.php
@@ -2,6 +2,15 @@
 
 namespace App\EasyAdmin;
 
-class VotesField
+use EasyCorp\Bundle\EasyAdminBundle\Contracts\Field\FieldInterface;
+use EasyCorp\Bundle\EasyAdminBundle\Field\FieldTrait;
+
+class VotesField implements FieldInterface
 {
+    use FieldTrait;
+
+    public static function new(string $propertyName, ?string $label = null)
+    {
+        // TODO: Implement new() method.
+    }
 }
