diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 9e29989..276fca2 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -35,6 +35,7 @@ class QuestionCrudController extends AbstractCrudController
     public function configureCrud(Crud $crud): Crud
     {
         return parent::configureCrud($crud)
+            ->overrideTemplate('crud/action', 'admin/crud/action.html.twig')
             ->setDefaultSort([
                 'askedBy.enabled' => 'DESC',
                 'createdAt' => 'DESC',
diff --git a/templates/admin/crud/action.html.twig b/templates/admin/crud/action.html.twig
new file mode 100644
index 0000000..d73054d
--- /dev/null
+++ b/templates/admin/crud/action.html.twig
@@ -0,0 +1 @@
+{{ dump(action) }}
