diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 85c62ce..76812e3 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -50,7 +50,8 @@ class QuestionCrudController extends AbstractCrudController
                 ];
             })
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-eye');
+            ->setIcon('fa fa-eye')
+            ->setLabel('View');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
