diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 749fec5..e37ab38 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -186,7 +186,7 @@ class QuestionCrudController extends AbstractCrudController
         return $this->redirect($targetUrl);
     }
 
-    public function export()
+    public function export(AdminContext $context)
     {
     }
 }
