diff --git a/src/EasyAdmin/VotesField.php b/src/EasyAdmin/VotesField.php
index e83c7fd2..5882ca7d 100644
--- a/src/EasyAdmin/VotesField.php
+++ b/src/EasyAdmin/VotesField.php
@@ -15,7 +15,10 @@ class VotesField implements FieldInterface
         return (new self())
             ->setProperty($propertyName)
             ->setLabel($label)
-            ->setTemplateName('crud/field/integer')
+            // this template is used in 'index' and 'detail' pages
+            ->setTemplatePath('admin/field/votes.html.twig')
+            // this is used in 'edit' and 'new' pages to edit the field contents
+            // you can use your own form types too
             ->setFormType(IntegerType::class)
             ->addCssClass('field-integer')
             ->setDefaultColumns('col-md-4 col-xxl-3');
