diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index a5fb230..55be36c 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -51,7 +51,8 @@ class QuestionCrudController extends AbstractCrudController
                 ];
             })
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-eye');
+            ->setIcon('fa fa-eye')
+            ->setLabel('View');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
