diff --git a/templates/admin/field/votes.html.twig b/templates/admin/field/votes.html.twig
index 727c07af..7431c5c6 100644
--- a/templates/admin/field/votes.html.twig
+++ b/templates/admin/field/votes.html.twig
@@ -1 +1,4 @@
-💯 votes!
+{# @var ea \EasyCorp\Bundle\EasyAdminBundle\Context\AdminContext #}
+{# @var field \EasyCorp\Bundle\EasyAdminBundle\Dto\FieldDto #}
+{# @var entity \EasyCorp\Bundle\EasyAdminBundle\Dto\EntityDto #}
+{{ field.formattedValue }} votes
