diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 0059ffa..3e6d9b0 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -44,7 +44,9 @@ class QuestionCrudController extends AbstractCrudController
     public function configureActions(Actions $actions): Actions
     {
         $viewAction = Action::new('view')
-            ->linkToRoute('app_question_show');
+            ->linkToRoute('app_question_show', static function ($param) {
+                dd($param);
+            });
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
