diff --git a/src/Controller/Admin/AnswerCrudController.php b/src/Controller/Admin/AnswerCrudController.php
index 370a858..614e153 100644
--- a/src/Controller/Admin/AnswerCrudController.php
+++ b/src/Controller/Admin/AnswerCrudController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller\Admin;
 
+use App\EasyAdmin\VotesField;
 use App\Entity\Answer;
 use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
 use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
@@ -25,6 +26,8 @@ class AnswerCrudController extends AbstractCrudController
             ->setTemplatePath('admin/field/votes.html.twig');
         yield AssociationField::new('question')
             ->hideOnIndex();
+        yield VotesField::new('votes', 'Total Votes')
+            ->setTextAlign('right');
         yield AssociationField::new('answeredBy');
         yield Field::new('createdAt')
             ->hideOnForm();
diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d5f4217..a35b683 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller\Admin;
 
+use App\EasyAdmin\VotesField;
 use App\Entity\Question;
 use Doctrine\ORM\QueryBuilder;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
@@ -32,7 +33,7 @@ class QuestionCrudController extends AbstractCrudController
         yield Field::new('name');
         yield TextareaField::new('question')
             ->hideOnIndex();
-        yield Field::new('votes', 'Total Votes')
+        yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right');
         yield AssociationField::new('askedBy')
             ->autocomplete()
