diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index facb3d2..23c88f3 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -58,6 +58,14 @@ class DashboardController extends AbstractDashboardController
             ]);
     }
 
+    public function configureCrud(): Crud
+    {
+        return parent::configureCrud()
+            ->setDefaultSort([
+                'id' => 'DESC',
+            ]);
+    }
+
     public function configureActions(): Actions
     {
         return parent::configureActions()
