diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index e87b8cd..4f0e91e 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -19,14 +19,18 @@ use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\UX\Chartjs\Builder\ChartBuilderInterface;
+use Symfony\UX\Chartjs\Model\Chart;
 
 class DashboardController extends AbstractDashboardController
 {
     private QuestionRepository $questionRepository;
+    private ChartBuilderInterface $chartBuilder;
 
-    public function __construct(QuestionRepository $questionRepository)
+    public function __construct(QuestionRepository $questionRepository, ChartBuilderInterface $chartBuilder)
     {
         $this->questionRepository = $questionRepository;
+        $this->chartBuilder = $chartBuilder;
     }
 
     #[IsGranted('ROLE_ADMIN')]
@@ -41,6 +45,7 @@ class DashboardController extends AbstractDashboardController
         return $this->render('admin/index.html.twig', [
             'latestQuestions' => $latestQuestions,
             'topVoted' => $topVoted,
+            'chart' => $this->createChart(),
         ]);
     }
 
@@ -91,4 +96,31 @@ class DashboardController extends AbstractDashboardController
         return parent::configureAssets()
             ->addWebpackEncoreEntry('admin');
     }
+
+    private function createChart(): Chart
+    {
+        $chart = $this->chartBuilder->createChart(Chart::TYPE_LINE);
+        $chart->setData([
+            'labels' => ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+            'datasets' => [
+                [
+                    'label' => 'My First dataset',
+                    'backgroundColor' => 'rgb(255, 99, 132)',
+                    'borderColor' => 'rgb(255, 99, 132)',
+                    'data' => [0, 10, 5, 2, 20, 30, 45],
+                ],
+            ],
+        ]);
+
+        $chart->setOptions([
+            'scales' => [
+                'y' => [
+                   'suggestedMin' => 0,
+                   'suggestedMax' => 100,
+                ],
+            ],
+        ]);
+
+        return $chart;
+    }
 }
diff --git a/templates/admin/index.html.twig b/templates/admin/index.html.twig
index be1baef..f1e87a1 100644
--- a/templates/admin/index.html.twig
+++ b/templates/admin/index.html.twig
@@ -27,5 +27,8 @@
                 {% endfor %}
             </ol>
         </div>
+        <div class="col-12">
+            {{ render_chart(chart) }}
+        </div>
     </div>
 {% endblock %}
