diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 65693991..b4e04981 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -126,4 +126,9 @@ class DashboardController extends AbstractDashboardController
 
         return $chart;
     }
+
+    public static function getSubscribedServices(): array
+    {
+        return parent::getSubscribedServices();
+    }
 }
