diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 9f800a5..95da11d 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -124,4 +124,9 @@ class DashboardController extends AbstractDashboardController
 
         return $chart;
     }
+
+    public static function getSubscribedServices(): array
+    {
+        return parent::getSubscribedServices();
+    }
 }
