diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 4f0e91e..0cee25b 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -25,18 +25,18 @@ use Symfony\UX\Chartjs\Model\Chart;
 class DashboardController extends AbstractDashboardController
 {
     private QuestionRepository $questionRepository;
-    private ChartBuilderInterface $chartBuilder;
 
-    public function __construct(QuestionRepository $questionRepository, ChartBuilderInterface $chartBuilder)
+    public function __construct(QuestionRepository $questionRepository)
     {
         $this->questionRepository = $questionRepository;
-        $this->chartBuilder = $chartBuilder;
     }
 
     #[IsGranted('ROLE_ADMIN')]
     #[Route('/admin', name: 'admin')]
-    public function index(): Response
+    public function index(ChartBuilderInterface $chartBuilder = null): Response
     {
+        assert(null !== $chartBuilder);
+
         $latestQuestions = $this->questionRepository
             ->findLatest();
         $topVoted = $this->questionRepository
@@ -45,7 +45,7 @@ class DashboardController extends AbstractDashboardController
         return $this->render('admin/index.html.twig', [
             'latestQuestions' => $latestQuestions,
             'topVoted' => $topVoted,
-            'chart' => $this->createChart(),
+            'chart' => $this->createChart($chartBuilder),
         ]);
     }
 
@@ -97,9 +97,9 @@ class DashboardController extends AbstractDashboardController
             ->addWebpackEncoreEntry('admin');
     }
 
-    private function createChart(): Chart
+    private function createChart(ChartBuilderInterface $chartBuilder): Chart
     {
-        $chart = $this->chartBuilder->createChart(Chart::TYPE_LINE);
+        $chart = $chartBuilder->createChart(Chart::TYPE_LINE);
         $chart->setData([
             'labels' => ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
             'datasets' => [
