diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 95c0f7b3..c565495f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -90,6 +90,8 @@ class QuestionCrudController extends AbstractCrudController
             });
         yield Field::new('createdAt')
             ->hideOnForm();
+        yield AssociationField::new('updatedBy')
+            ->onlyOnDetail();
     }
 
     public function configureFilters(Filters $filters): Filters
