diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 655e9a9..81d9525 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -89,6 +89,8 @@ class QuestionCrudController extends AbstractCrudController
             });
         yield Field::new('createdAt')
             ->hideOnForm();
+        yield AssociationField::new('updatedBy')
+            ->onlyOnDetail();
     }
 
     public function configureFilters(Filters $filters): Filters
