diff --git a/src/EventSubscriber/BlameableSubscriber.php b/src/EventSubscriber/BlameableSubscriber.php
index 12c93d5..a809245 100644
--- a/src/EventSubscriber/BlameableSubscriber.php
+++ b/src/EventSubscriber/BlameableSubscriber.php
@@ -36,7 +36,7 @@ class BlameableSubscriber implements EventSubscriberInterface
     public static function getSubscribedEvents()
     {
         return [
-            BeforeEntityUpdatedEvent::class => 'onBeforeEntityUpdatedEvent',
+            //BeforeEntityUpdatedEvent::class => 'onBeforeEntityUpdatedEvent',
         ];
     }
 }
