diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 655e9a9..81d9525 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -89,6 +89,8 @@ class QuestionCrudController extends AbstractCrudController
             });
         yield Field::new('createdAt')
             ->hideOnForm();
+        yield AssociationField::new('updatedBy')
+            ->onlyOnDetail();
     }
 
     public function configureFilters(Filters $filters): Filters
diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index 02f18ae..2a2643f 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -66,6 +66,12 @@ class Question
      */
     private $isApproved = false;
 
+    /**
+     * @ORM\ManyToOne(targetEntity=User::class)
+     * @ORM\JoinColumn(nullable=true)
+     */
+    private $updatedBy;
+
     public function __construct()
     {
         $this->answers = new ArrayCollection();
@@ -208,4 +214,14 @@ class Question
     {
         $this->isApproved = $isApproved;
     }
+
+    public function getUpdatedBy(): ?User
+    {
+        return $this->updatedBy;
+    }
+
+    public function setUpdatedBy(User $updatedBy): void
+    {
+        $this->updatedBy = $updatedBy;
+    }
 }
