diff --git a/migrations/Version20211102150046.php b/migrations/Version20211102150046.php
new file mode 100644
index 0000000..0683e0f
--- /dev/null
+++ b/migrations/Version20211102150046.php
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20211102150046 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question ADD updated_by_id INT DEFAULT NULL');
+        $this->addSql('ALTER TABLE question ADD CONSTRAINT FK_B6F7494E896DBBDE FOREIGN KEY (updated_by_id) REFERENCES user (id)');
+        $this->addSql('CREATE INDEX IDX_B6F7494E896DBBDE ON question (updated_by_id)');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question DROP FOREIGN KEY FK_B6F7494E896DBBDE');
+        $this->addSql('DROP INDEX IDX_B6F7494E896DBBDE ON question');
+        $this->addSql('ALTER TABLE question DROP updated_by_id');
+    }
+}
diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 655e9a9..81d9525 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -89,6 +89,8 @@ class QuestionCrudController extends AbstractCrudController
             });
         yield Field::new('createdAt')
             ->hideOnForm();
+        yield AssociationField::new('updatedBy')
+            ->onlyOnDetail();
     }
 
     public function configureFilters(Filters $filters): Filters
diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index 314e739..e5c8c9c 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -66,6 +66,12 @@ class Question
      */
     private $isApproved = false;
 
+    /**
+     * @ORM\ManyToOne(targetEntity=User::class)
+     * @ORM\JoinColumn(nullable=true)
+     */
+    private $updatedBy;
+
     public function __construct()
     {
         $this->answers = new ArrayCollection();
@@ -213,4 +219,14 @@ class Question
     {
         $this->isApproved = $isApproved;
     }
+
+    public function getUpdatedBy(): ?User
+    {
+        return $this->updatedBy;
+    }
+
+    public function setUpdatedBy(User $updatedBy): void
+    {
+        $this->updatedBy = $updatedBy;
+    }
 }
