diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d6da96b..04c651b 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -27,7 +27,7 @@ class QuestionCrudController extends AbstractCrudController
             ->hideOnIndex()
             ->setFormTypeOptions([
                 'attr' => [
-                    'readonly' => $pageName !== Action::NEW,
+                    'disabled' => $pageName !== Action::NEW,
                 ],
             ]);
         yield Field::new('name');
