diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index baa1c4f5..475110c0 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -35,6 +35,7 @@ class UserCrudController extends AbstractCrudController
             ->renderAsSwitch(false);
         yield DateField::new('createdAt')
             ->hideOnForm();
-        yield ArrayField::new('roles');
+        yield ArrayField::new('roles')
+            ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER');
     }
 }
