diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 5bf2bdc..37c349c 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,8 @@ class UserCrudController extends AbstractCrudController
             AvatarField::new('avatar'),
             TextField::new('email'),
             TextField::new('fullName'),
-            BooleanField::new('enabled'),
+            BooleanField::new('enabled')
+                ->renderAsSwitch(false),
             DateField::new('createdAt'),
         ];
     }
