diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index a720663..b5c61fa 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -26,6 +26,7 @@ class QuestionCrudController extends AbstractCrudController
             ->hideOnIndex();
         yield Field::new('votes', 'Total Votes')
             ->setTextAlign('right');
+        yield AssociationField::new('askedBy');
         yield Field::new('createdAt')
             ->hideOnForm();
     }
