diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index de52423e..72c14cf5 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,6 +25,8 @@ class UserCrudController extends AbstractCrudController
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
+        yield Field::new('firstName');
+        yield Field::new('lastName');
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
