diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 86f22b0..6d8f07f 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,8 @@ class UserCrudController extends AbstractCrudController
         yield AvatarField::new('avatar');
         yield TextField::new('email');
         yield TextField::new('fullName');
-        yield BooleanField::new('enabled');
+        yield BooleanField::new('enabled')
+            ->renderAsSwitch(false);
         yield DateField::new('createdAt');
     }
 }
