diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 880ddb3..2ca48e8 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,7 @@ class UserCrudController extends AbstractCrudController
             ->onlyOnIndex();
         yield EmailField::new('email');
         yield TextField::new('fullName')
-            ->onlyOnIndex();
+            ->hideOnForm();
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
