diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 0649ed8..743f946 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,8 @@ class UserCrudController extends AbstractCrudController
             ->onlyOnIndex();
         yield AvatarField::new('avatar');
         yield TextField::new('email');
-        yield TextField::new('fullName');
+        yield TextField::new('fullName')
+            ->setHelp('First, then Last');
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
