diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 125f090e..2d44a2e2 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -34,6 +34,7 @@ class UserCrudController extends AbstractCrudController
             ->renderAsSwitch(false);
         yield DateField::new('createdAt')
             ->hideOnForm();
-        yield ArrayField::new('roles');
+        yield ArrayField::new('roles')
+            ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER');
     }
 }
