diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 29bfc7c..86f22b0 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -22,7 +22,7 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id');
         yield AvatarField::new('avatar');
-        yield EmailField::new('email');
+        yield TextField::new('email');
         yield TextField::new('fullName');
         yield BooleanField::new('enabled');
         yield DateField::new('createdAt');
