diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 0c8f240a..de52423e 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,7 @@ class UserCrudController extends AbstractCrudController
             ->onlyOnIndex();
         yield EmailField::new('email');
         yield TextField::new('fullName')
-            ->onlyOnIndex();
+            ->hideOnForm();
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
