diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 77c3d2f..48d3a80 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -33,6 +33,7 @@ class UserCrudController extends AbstractCrudController
             ->onlyOnForms();
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
-        yield DateField::new('createdAt');
+        yield DateField::new('createdAt')
+            ->hideOnForm();
     }
 }
