diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 42d84b00..3439fb65 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -26,8 +26,10 @@ class UserCrudController extends AbstractCrudController
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
-        yield Field::new('firstName');
-        yield Field::new('lastName');
+        yield Field::new('firstName')
+            ->onlyOnForms();
+        yield Field::new('lastName')
+            ->onlyOnForms();
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
