diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 2d44a2e2..240d0b49 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -11,6 +11,7 @@ use EasyCorp\Bundle\EasyAdminBundle\Field\EmailField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\Field;
 use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 
 class UserCrudController extends AbstractCrudController
 {
@@ -34,7 +35,14 @@ class UserCrudController extends AbstractCrudController
             ->renderAsSwitch(false);
         yield DateField::new('createdAt')
             ->hideOnForm();
+
+        $roles = ['ROLE_SUPER_ADMIN', 'ROLE_ADMIN', 'ROLE_MODERATOR', 'ROLE_USER'];
         yield ArrayField::new('roles')
-            ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER');
+            ->setFormType(ChoiceType::class)
+            ->setFormTypeOptions([
+                'choices' => array_combine($roles, $roles),
+                'multiple' => true,
+                'expanded' => true,
+            ]);
     }
 }
