diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index b5d6b3c..77c3d2f 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -27,8 +27,10 @@ class UserCrudController extends AbstractCrudController
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
-        yield Field::new('firstName');
-        yield Field::new('lastName');
+        yield Field::new('firstName')
+            ->onlyOnForms();
+        yield Field::new('lastName')
+            ->onlyOnForms();
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
