diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 1367558..b5d6b3c 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -27,6 +27,8 @@ class UserCrudController extends AbstractCrudController
         yield EmailField::new('email');
         yield TextField::new('fullName')
             ->hideOnForm();
+        yield Field::new('firstName');
+        yield Field::new('lastName');
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt');
