diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 219789b..95c0f7b 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -97,6 +97,7 @@ class QuestionCrudController extends AbstractCrudController
         return parent::configureFilters($filters)
             ->add('topic')
             ->add('createdAt')
-            ->add('votes');
+            ->add('votes')
+            ->add('name');
     }
 }
