diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 69a2e4ad..219789b8 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -96,6 +96,7 @@ class QuestionCrudController extends AbstractCrudController
     {
         return parent::configureFilters($filters)
             ->add('topic')
-            ->add('createdAt');
+            ->add('createdAt')
+            ->add('votes');
     }
 }
