diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 013d61d..952624b 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -94,6 +94,7 @@ class QuestionCrudController extends AbstractCrudController
         return parent::configureFilters($filters)
             ->add('topic')
             ->add('createdAt')
-            ->add('votes');
+            ->add('votes')
+            ->add('name');
     }
 }
