diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 0124be3..7e3dca6 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -87,6 +87,6 @@ class UserCrudController extends AbstractCrudController
     public function configureFilters(Filters $filters): Filters
     {
         return parent::configureFilters($filters)
-            ->add(BooleanFilter::new('enabled'));
+            ->add(BooleanFilter::new('enabled')->setFormTypeOption('expanded', false));
     }
 }
