diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index b5b51fb..94e6270 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -70,6 +70,12 @@ class UserCrudController extends AbstractCrudController
     {
         return parent::configureFilters($filters)
             ->add('enabled')
-            ->add(ArrayFilter::new('roles'));
+            ->add(ArrayFilter::new('roles')
+                ->setChoices([
+                    'Super Admin' => 'ROLE_SUPER_ADMIN',
+                    'Admin' => 'ROLE_ADMIN',
+                    'Moderator' => 'ROLE_MODERATOR',
+                ])
+            );
     }
 }
