diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 6447cfd..ea8a24e 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -93,6 +93,7 @@ class QuestionCrudController extends AbstractCrudController
 
     public function configureFilters(Filters $filters): Filters
     {
-        return parent::configureFilters($filters);
+        return parent::configureFilters($filters)
+            ->add('topic');
     }
 }
