diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index c5d98e6..6d4243d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -95,6 +95,7 @@ class QuestionCrudController extends AbstractCrudController
     {
         return parent::configureFilters($filters)
             ->add('topic')
-            ->add('createdAt');
+            ->add('createdAt')
+            ->add('votes');
     }
 }
