diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 25bce7f..8a31427 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -96,6 +96,7 @@ class QuestionCrudController extends AbstractCrudController
         return parent::configureFilters($filters)
             ->add('topic')
             ->add('createdAt')
-            ->add('votes');
+            ->add('votes')
+            ->add('name');
     }
 }
