diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 930aa00f..28c83fb2 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -36,7 +36,15 @@
                             </div>
                         </div>
                         <div class="col">
-                            <h1 class="q-title-show">{{ question.name }}</h1>
+                            <div class="d-flex justify-content-between">
+                                <h1 class="q-title-show">{{ question.name }}</h1>
+
+                                {% if is_granted('ROLE_ADMIN') %}
+                                    <a class="text-white" href="">
+                                        <span class="fa fa-edit"></span>
+                                    </a>
+                                {% endif %}
+                            </div>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
                                 <p class="d-inline">{{ question.question|parse_markdown }}</p>
