diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 28c83fb2..1e73dfc7 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -40,7 +40,11 @@
                                 <h1 class="q-title-show">{{ question.name }}</h1>
 
                                 {% if is_granted('ROLE_ADMIN') %}
-                                    <a class="text-white" href="">
+                                <a class="text-white" href="{{ ea_url()
+                                    .setController('App\\Controller\\Admin\\QuestionCrudController')
+                                    .setAction('edit')
+                                    .setEntityId(question.id)
+                                }}">
                                         <span class="fa fa-edit"></span>
                                     </a>
                                 {% endif %}
