diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5505235..8ec7074 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -35,6 +35,8 @@ class QuestionCrudController extends AbstractCrudController
 
     public function configureActions(Actions $actions): Actions
     {
+        $viewAction = Action::new('view');
+
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
                 $action->displayIf(static function (Question $question) {
@@ -48,7 +50,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission(Action::EDIT, 'ROLE_MODERATOR')
             ->setPermission(Action::NEW, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::DELETE, 'ROLE_SUPER_ADMIN')
-            ->add(Crud::PAGE_DETAIL, 'view');
+            ->add(Crud::PAGE_DETAIL, $viewAction);
     }
 
     public function configureFields(string $pageName): iterable
