diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 1ca2560..e6f647e 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -40,7 +40,8 @@ class QuestionCrudController extends AbstractCrudController
                 return $this->generateUrl('app_question_show', [
                     'slug' => $question->getSlug(),
                 ]);
-            });
+            })
+            ->addCssClass('btn btn-success');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
