diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 220be47..3c49755 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -36,7 +36,10 @@ class QuestionCrudController extends AbstractCrudController
 
     public function configureActions(Actions $actions): Actions
     {
-        $viewAction = Action::new('view');
+        $viewAction = Action::new('view')
+            ->linkToUrl(function () {
+                dd(func_get_args());
+            });
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
