diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 7ba64e3..3cff711 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -48,7 +48,8 @@ class QuestionCrudController extends AbstractCrudController
                 return [
                     'slug' => $question->getSlug(),
                 ];
-            });
+            })
+            ->addCssClass('btn btn-success');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
