diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index f56edcb5..2af650b2 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -42,7 +42,8 @@ class QuestionCrudController extends AbstractCrudController
                     'slug' => $question->getSlug(),
                 ]);
             })
-            ->addCssClass('btn btn-success');
+            ->addCssClass('btn btn-success')
+            ->setIcon('fa fa-eye');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
