diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 57acbd8..83b7536 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -37,8 +37,10 @@ class QuestionCrudController extends AbstractCrudController
     public function configureActions(Actions $actions): Actions
     {
         $viewAction = Action::new('view')
-            ->linkToUrl(function () {
-                dd(func_get_args());
+            ->linkToUrl(function (Question $question) {
+                return $this->generateUrl('app_question_show', [
+                    'slug' => $question->getSlug(),
+                ]);
             });
 
         return parent::configureActions($actions)
