diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 8235c2e..b18086a 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -43,7 +43,8 @@ class QuestionCrudController extends AbstractCrudController
                 ]);
             })
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-eye');
+            ->setIcon('fa fa-eye')
+            ->setLabel('View');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
