diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 813dd46..cf4ed9d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -42,7 +42,8 @@ class QuestionCrudController extends AbstractCrudController
                 ]);
             })
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-eye');
+            ->setIcon('fa fa-eye')
+            ->setLabel('View');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
