diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index c9b37e7e..bfb2b97e 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -38,8 +38,10 @@ class QuestionCrudController extends AbstractCrudController
     public function configureActions(Actions $actions): Actions
     {
         $viewAction = Action::new('view')
-            ->linkToUrl(function() {
-                dd(func_get_args());
+            ->linkToUrl(function(Question $question) {
+                return $this->generateUrl('app_question_show', [
+                    'slug' => $question->getSlug(),
+                ]);
             });
 
         return parent::configureActions($actions)
