diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d0be5b4..4463fb2 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -42,7 +42,8 @@ class QuestionCrudController extends AbstractCrudController
                 ]);
             })
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-eye');
+            ->setIcon('fa fa-eye')
+            ->setLabel('View on site');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
