diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index eb5584a..d0be5b4 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -41,7 +41,8 @@ class QuestionCrudController extends AbstractCrudController
                     'slug' => $question->getSlug(),
                 ]);
             })
-            ->addCssClass('btn btn-success');
+            ->addCssClass('btn btn-success')
+            ->setIcon('fa fa-eye');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
