diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 8fa3308..f81b99f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -63,7 +63,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission(Action::NEW, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::DELETE, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::BATCH_DELETE, 'ROLE_SUPER_ADMIN')
-            ->add(Crud::PAGE_DETAIL, $viewAction);
+            ->add(Crud::PAGE_DETAIL, $viewAction)
+            ->add(Crud::PAGE_INDEX, $viewAction);
     }
 
     public function configureFields(string $pageName): iterable
