diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5928d2f..dbcb64a 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -40,7 +40,8 @@ class QuestionCrudController extends AbstractCrudController
                 return $this->generateUrl('app_question_show', [
                     'slug' => $question->getSlug(),
                 ]);
-            });
+            })
+            ->addCssClass('btn btn-success');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
