diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index a091293..56e5ff1 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -43,7 +43,8 @@ class QuestionCrudController extends AbstractCrudController
                     'slug' => $question->getSlug(),
                 ]);
             })
-            ->addCssClass('btn btn-success');
+            ->addCssClass('btn btn-success')
+            ->setIcon('fa fa-eye');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
