diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 21097ff..a1877b8 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -176,7 +176,7 @@ class QuestionCrudController extends AbstractCrudController
         return $this->redirect($targetUrl);
     }
 
-    public function export()
+    public function export(AdminContext $context)
     {
     }
 }
