diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 799216f4..18ddd0d9 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -101,7 +101,14 @@ class QuestionCrudController extends AbstractCrudController
             ->add(Crud::PAGE_DETAIL, $viewAction)
             ->add(Crud::PAGE_INDEX, $viewAction)
             ->add(Crud::PAGE_DETAIL, $approveAction)
-            ->add(Crud::PAGE_INDEX, $exportAction);
+            ->add(Crud::PAGE_INDEX, $exportAction)
+            ->reorder(Crud::PAGE_DETAIL, [
+                'approve',
+                'view',
+                Action::EDIT,
+                Action::INDEX,
+                Action::DELETE,
+            ]);
     }
 
     public function configureFields(string $pageName): iterable
