diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d477828..cc5d72d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -188,4 +188,8 @@ class QuestionCrudController extends AbstractCrudController
 
         return $this->redirect($targetUrl);
     }
+
+    public function export()
+    {
+    }
 }
