diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 3d01d32..291afc5 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -58,7 +58,8 @@ class QuestionCrudController extends AbstractCrudController
         $exportAction = Action::new('export')
             ->linkToCrudAction('export')
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-download');
+            ->setIcon('fa fa-download')
+            ->createAsGlobalAction();
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
