diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 637e4d5..a2e9f8f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -55,6 +55,10 @@ class QuestionCrudController extends AbstractCrudController
             ->displayIf(static function (Question $question): bool {
                 return !$question->getIsApproved();
             });
+        $exportAction = Action::new('export')
+            ->linkToCrudAction('export')
+            ->addCssClass('btn btn-success')
+            ->setIcon('fa fa-download');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
@@ -70,7 +74,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission(Action::NEW, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::DELETE, 'ROLE_SUPER_ADMIN')
             ->add(Crud::PAGE_DETAIL, $viewAction)
-            ->add(Crud::PAGE_DETAIL, $approveAction);
+            ->add(Crud::PAGE_DETAIL, $approveAction)
+            ->add(Crud::PAGE_INDEX, $exportAction);
     }
 
     public function configureFields(string $pageName): iterable
