diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 6bdadbb..dc89aa4 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -175,4 +175,8 @@ class QuestionCrudController extends AbstractCrudController
 
         return $this->redirect($targetUrl);
     }
+
+    public function export()
+    {
+    }
 }
