diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 8038d29..d37720f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -59,7 +59,8 @@ class QuestionCrudController extends AbstractCrudController
         $exportAction = Action::new('export')
             ->linkToCrudAction('export')
             ->addCssClass('btn btn-success')
-            ->setIcon('fa fa-download');
+            ->setIcon('fa fa-download')
+            ->createAsGlobalAction();
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
