diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 725ce3a..99d72da 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -182,4 +182,8 @@ class QuestionCrudController extends AbstractCrudController
 
         return $this->redirect($targetUrl);
     }
+
+    public function export()
+    {
+    }
 }
