diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 83e60aa..a0b9ff8 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -186,4 +186,8 @@ class QuestionCrudController extends AbstractCrudController
 
         return $this->redirect($targetUrl);
     }
+
+    public function export()
+    {
+    }
 }
