diff --git a/src/EventSubscriber/HideActionSubscriber.php b/src/EventSubscriber/HideActionSubscriber.php
new file mode 100644
index 0000000..bab63f8
--- /dev/null
+++ b/src/EventSubscriber/HideActionSubscriber.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use EasyCorp\Bundle\EasyAdminBundle\Event\BeforeCrudActionEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class HideActionSubscriber implements EventSubscriberInterface
+{
+    public function onBeforeCrudActionEvent(BeforeCrudActionEvent $event)
+    {
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            BeforeCrudActionEvent::class => 'onBeforeCrudActionEvent',
+        ];
+    }
+}
