diff --git a/src/EventSubscriber/HideActionSubscriber.php b/src/EventSubscriber/HideActionSubscriber.php
index bab63f8..1511df0 100644
--- a/src/EventSubscriber/HideActionSubscriber.php
+++ b/src/EventSubscriber/HideActionSubscriber.php
@@ -9,6 +9,7 @@ class HideActionSubscriber implements EventSubscriberInterface
 {
     public function onBeforeCrudActionEvent(BeforeCrudActionEvent $event)
     {
+        dd($event);
     }
 
     public static function getSubscribedEvents()
