diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index ea4d0d6..3a80013 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -132,6 +132,10 @@ class QuestionCrudController extends AbstractCrudController
      */
     public function deleteEntity(EntityManagerInterface $entityManager, $entityInstance): void
     {
+        if ($entityInstance->getIsApproved()) {
+            throw new \Exception('Deleting approved questions is forbidden!');
+        }
+
         parent::deleteEntity($entityManager, $entityInstance);
     }
 }
