diff --git a/src/EventSubscriber/HideActionSubscriber.php b/src/EventSubscriber/HideActionSubscriber.php
index bab63f8..a83145c 100644
--- a/src/EventSubscriber/HideActionSubscriber.php
+++ b/src/EventSubscriber/HideActionSubscriber.php
@@ -2,6 +2,7 @@
 
 namespace App\EventSubscriber;
 
+use App\Entity\Question;
 use EasyCorp\Bundle\EasyAdminBundle\Event\BeforeCrudActionEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
@@ -9,6 +10,15 @@ class HideActionSubscriber implements EventSubscriberInterface
 {
     public function onBeforeCrudActionEvent(BeforeCrudActionEvent $event)
     {
+        if (!$adminContext = $event->getAdminContext()) {
+            return;
+        }
+        if (!$crudDto = $adminContext->getCrud()) {
+            return;
+        }
+        if ($crudDto->getEntityFqcn() !== Question::class) {
+            return;
+        }
     }
 
     public static function getSubscribedEvents()
