diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 2e05c5b..5935f72 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -131,6 +131,10 @@ class QuestionCrudController extends AbstractCrudController
      */
     public function deleteEntity(EntityManagerInterface $entityManager, $entityInstance): void
     {
+        if ($entityInstance->getIsApproved()) {
+            throw new \Exception('Deleting approved questions is forbidden!');
+        }
+
         parent::deleteEntity($entityManager, $entityInstance);
     }
 }
