diff --git a/src/EventSubscriber/HideActionSubscriber.php b/src/EventSubscriber/HideActionSubscriber.php
index 68ab4e3..c85ab80 100644
--- a/src/EventSubscriber/HideActionSubscriber.php
+++ b/src/EventSubscriber/HideActionSubscriber.php
@@ -27,6 +27,9 @@ class HideActionSubscriber implements EventSubscriberInterface
             foreach ($entities as $entityDto) {
                 $this->hideDeleteActionIfApprovedQuestion($entityDto);
             }
+        } else {
+            $entityDto = $event->getResponseParameters()->get('entity');
+            $this->hideDeleteActionIfApprovedQuestion($entityDto);
         }
     }
 
