diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 735a17f..e688391 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -136,7 +136,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
             ->collapsible()
-            ->setIcon('fa fa-info');
+            ->setIcon('fa fa-info')
+            ->setHelp('Additional Details');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(static function ($value, ?Question $question): ?string {
