diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 2ca3a4c..d796d1f 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -130,7 +130,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
-            ->collapsible();
+            ->collapsible()
+            ->setIcon('fa fa-info');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
