diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index bfc9b96..e0a4d70 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -129,7 +129,8 @@ class QuestionCrudController extends AbstractCrudController
         yield Field::new('votes', 'Total Votes')
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
-        yield FormField::addPanel('Details');
+        yield FormField::addPanel('Details')
+            ->collapsible();
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
