diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 2862ed5..f192e54 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -117,14 +117,14 @@ class QuestionCrudController extends AbstractCrudController
         yield FormField::addTab('Basic Data')
             ->collapsible();
         yield AssociationField::new('topic');
+        yield Field::new('name')
+            ->setSortable(false);
         yield Field::new('slug')
             ->hideOnIndex()
             ->setFormTypeOption(
                 'disabled',
                 $pageName !== Crud::PAGE_NEW
             );
-        yield Field::new('name')
-            ->setSortable(false);
         yield TextareaField::new('question')
             ->hideOnIndex()
             ->setFormTypeOptions([
