diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5a5a102a..6bc3a485 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -144,7 +144,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
             ->collapsible()
-            ->setIcon('fa fa-info');
+            ->setIcon('fa fa-info')
+            ->setHelp('Additional Details');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(static function ($value, ?Question $question): ?string {
