diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5489a85..2f9ae53 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -132,7 +132,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
             ->collapsible()
-            ->setIcon('fa fa-info');
+            ->setIcon('fa fa-info')
+            ->setHelp('Additional Details');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
