diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index f973580..5e55411 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -137,8 +137,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addTab('Details')
-            ->collapsible()
-            ->setIcon('fa fa-info')
+            ->setIcon('info')
             ->setHelp('Additional Details');
         yield AssociationField::new('askedBy')
             ->autocomplete()
