diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5b578f9e..f7d701c4 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -120,13 +120,15 @@ class QuestionCrudController extends AbstractCrudController
             ->collapsible();
         yield AssociationField::new('topic');
         yield Field::new('name')
-            ->setSortable(false);
+            ->setSortable(false)
+            ->setColumns(5);
         yield Field::new('slug')
             ->hideOnIndex()
             ->setFormTypeOption(
                 'disabled',
                 $pageName !== Crud::PAGE_NEW
-            );
+            )
+            ->setColumns(5);
         yield TextareaField::new('question')
             ->hideOnIndex()
             ->setFormTypeOptions([
