diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index f5f3874..3d6a0df 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -48,9 +48,11 @@ class UserCrudController extends AbstractCrudController
         yield TextField::new('fullName')
             ->hideOnForm();
         yield Field::new('firstName')
-            ->onlyOnForms();
+            ->onlyOnForms()
+            ->setColumns(3);
         yield Field::new('lastName')
-            ->onlyOnForms();
+            ->onlyOnForms()
+            ->setColumns(3);
         yield BooleanField::new('enabled')
             ->renderAsSwitch(false);
         yield DateField::new('createdAt')
