diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index dbacf4e6..b0c2f693 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -116,6 +116,7 @@ class QuestionCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
+        yield FormField::addPanel('Basic Data');
         yield AssociationField::new('topic');
         yield Field::new('slug')
             ->hideOnIndex()
