diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 3c357c5..6232f86 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -114,7 +114,8 @@ class QuestionCrudController extends AbstractCrudController
     {
         yield IdField::new('id')
             ->onlyOnIndex();
-        yield FormField::addPanel('Basic Data');
+        yield FormField::addPanel('Basic Data')
+            ->collapsible();
         yield AssociationField::new('topic');
         yield Field::new('slug')
             ->hideOnIndex()
@@ -139,7 +140,8 @@ class QuestionCrudController extends AbstractCrudController
         yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
-        yield FormField::addPanel('Details');
+        yield FormField::addPanel('Details')
+            ->collapsible();
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(static function ($value, ?Question $question): ?string {
