diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 65dec3a..f8cb380 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -128,7 +128,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
-            ->collapsible();
+            ->collapsible()
+            ->setIcon('fa fa-info');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(static function ($value, ?Question $question): ?string {
