diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 880dcba..27b051d 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -126,7 +126,8 @@ class QuestionCrudController extends AbstractCrudController
         yield Field::new('votes', 'Total Votes')
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
-        yield FormField::addPanel('Details');
+        yield FormField::addPanel('Details')
+            ->collapsible();
         yield Field::new('askedAt');
         yield AssociationField::new('askedBy')
             ->autocomplete()
