diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index adae902..22b8c11 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -133,7 +133,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setTextAlign('right')
             ->setPermission('ROLE_SUPER_ADMIN');
         yield FormField::addPanel('Details')
-            ->collapsible();
+            ->collapsible()
+            ->setIcon('fa fa-info');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
