diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 79b9cf4b..4a3fbe8c 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -57,7 +57,7 @@ class DashboardController extends AbstractDashboardController
         yield MenuItem::linktoDashboard('Dashboard', 'fa fa-dashboard');
         yield MenuItem::subMenu('Questions', 'fa fa-question-circle')
             ->setSubItems([
-                MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
+                MenuItem::linkToCrud('All', 'fa fa-question-circle', Question::class)
                     ->setController(QuestionCrudController::class)
                     ->setPermission('ROLE_MODERATOR'),
                 MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
