diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 4b410ea..0d6841e 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -58,11 +58,14 @@ class DashboardController extends AbstractDashboardController
     public function configureMenuItems(): iterable
     {
         yield MenuItem::linkToDashboard('Dashboard', 'fa fa-dashboard');
-        yield MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
-            ->setController(QuestionCrudController::class)
-            ->setPermission('ROLE_MODERATOR');
-        yield MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
-            ->setController(QuestionPendingApprovalCrudController::class);
+        yield MenuItem::subMenu('Questions', 'fa fa-question-circle')
+            ->setSubItems([
+                MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
+                    ->setController(QuestionCrudController::class)
+                    ->setPermission('ROLE_MODERATOR'),
+                MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
+                    ->setController(QuestionPendingApprovalCrudController::class),
+            ]);
         yield MenuItem::linkToCrud('Answers', 'fas fa-comments', Answer::class);
         yield MenuItem::linkToCrud('Topics', 'fas fa-folder', Topic::class);
         yield MenuItem::linkToCrud('Users', 'fas fa-users', User::class);
