diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index df0332d2..f2778fe7 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -22,6 +22,8 @@ class QuestionCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->onlyOnIndex();
         yield AssociationField::new('topic');
+        yield Field::new('slug')
+            ->hideOnIndex();
         yield Field::new('name');
         yield TextareaField::new('question')
             ->hideOnIndex();
