diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 0d393c2..ae5bce4 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -62,6 +62,7 @@ class DashboardController extends AbstractDashboardController
     {
         yield MenuItem::linkToDashboard('Dashboard', 'fa fa-dashboard');
         yield MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
+            ->setController(QuestionCrudController::class)
             ->setPermission('ROLE_MODERATOR');
         yield MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
             ->setController(QuestionPendingApprovalCrudController::class);
