diff --git a/src/Controller/Admin/QuestionPendingApprovalCrudController.php b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
index 8e4a56e4..1050c81f 100644
--- a/src/Controller/Admin/QuestionPendingApprovalCrudController.php
+++ b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
@@ -18,7 +18,8 @@ class QuestionPendingApprovalCrudController extends QuestionCrudController
             ->setPageTitle(Crud::PAGE_INDEX, 'Questions pending approval')
             ->setPageTitle(Crud::PAGE_DETAIL, static function (Question $question) {
                 return sprintf('#%s %s', $question->getId(), $question->getName());
-            });
+            })
+            ->setHelp(Crud::PAGE_INDEX, 'Questions that are not published to users until approved by moderators');
     }
 
     public function createIndexQueryBuilder(SearchDto $searchDto, EntityDto $entityDto, FieldCollection $fields, FilterCollection $filters): QueryBuilder
