diff --git a/src/Controller/Admin/QuestionPendingApprovalCrudController.php b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
index 44a81e8..ff54903 100644
--- a/src/Controller/Admin/QuestionPendingApprovalCrudController.php
+++ b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
@@ -12,6 +12,8 @@ class QuestionPendingApprovalCrudController extends QuestionCrudController
 {
     public function createIndexQueryBuilder(SearchDto $searchDto, EntityDto $entityDto, FieldCollection $fields, FilterCollection $filters): QueryBuilder
     {
-        return parent::createIndexQueryBuilder($searchDto, $entityDto, $fields, $filters);
+        return parent::createIndexQueryBuilder($searchDto, $entityDto, $fields, $filters)
+            ->andWhere('entity.isApproved = :approved')
+            ->setParameter('approved', false);
     }
 }
