diff --git a/src/Controller/Admin/AnswerCrudController.php b/src/Controller/Admin/AnswerCrudController.php
index 9094bbf..9faf999 100644
--- a/src/Controller/Admin/AnswerCrudController.php
+++ b/src/Controller/Admin/AnswerCrudController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller\Admin;
 
+use App\Controller\QuestionController;
 use App\EasyAdmin\VotesField;
 use App\Entity\Answer;
 use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
@@ -26,6 +27,7 @@ class AnswerCrudController extends AbstractCrudController
             ->setTemplatePath('admin/field/votes.html.twig');
         yield AssociationField::new('question')
             ->autocomplete()
+            ->setCrudController(QuestionController::class)
             ->hideOnIndex();
         yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right');
