diff --git a/src/Controller/Admin/QuestionPendingApprovalCrudController.php b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
index cdd5408..0044352 100644
--- a/src/Controller/Admin/QuestionPendingApprovalCrudController.php
+++ b/src/Controller/Admin/QuestionPendingApprovalCrudController.php
@@ -14,7 +14,8 @@ class QuestionPendingApprovalCrudController extends QuestionCrudController
     public function configureCrud(Crud $crud): Crud
     {
         return parent::configureCrud($crud)
-            ->setPageTitle(Crud::PAGE_INDEX, 'Questions pending approval');
+            ->setPageTitle(Crud::PAGE_INDEX, 'Questions pending approval')
+            ->setHelp(Crud::PAGE_INDEX, 'Questions that are not published to users until approved by moderators');
     }
 
     public function createIndexQueryBuilder(SearchDto $searchDto, EntityDto $entityDto, FieldCollection $fields, FilterCollection $filters): QueryBuilder
