diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index cdb70b0..c3b979c 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -56,6 +56,7 @@ class DashboardController extends AbstractDashboardController
     {
         yield MenuItem::linktoDashboard('Dashboard', 'fa fa-dashboard');
         yield MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
+            ->setController(QuestionCrudController::class)
             ->setPermission('ROLE_MODERATOR');
         yield MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
             ->setController(QuestionPendingApprovalCrudController::class);
