diff --git a/src/Controller/Admin/AnswerCrudController.php b/src/Controller/Admin/AnswerCrudController.php
index 614e1533..9094bbfc 100644
--- a/src/Controller/Admin/AnswerCrudController.php
+++ b/src/Controller/Admin/AnswerCrudController.php
@@ -25,6 +25,7 @@ class AnswerCrudController extends AbstractCrudController
         yield IntegerField::new('votes')
             ->setTemplatePath('admin/field/votes.html.twig');
         yield AssociationField::new('question')
+            ->autocomplete()
             ->hideOnIndex();
         yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right');
