diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index ed43cb5..b5b3e55 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -59,6 +59,7 @@ class DashboardController extends AbstractDashboardController
     {
         yield MenuItem::linkToDashboard('Dashboard', 'fa fa-dashboard');
         yield MenuItem::linkToCrud('Questions', 'fa fa-question-circle', Question::class)
+            ->setController(QuestionCrudController::class)
             ->setPermission('ROLE_MODERATOR');
         yield MenuItem::linkToCrud('Pending Approval', 'far fa-question-circle', Question::class)
             ->setPermission('ROLE_MODERATOR')
