diff --git a/templates/admin/crud/field/id_with_icon.html.twig b/templates/admin/crud/field/id_with_icon.html.twig
index c624098..0e970bd 100644
--- a/templates/admin/crud/field/id_with_icon.html.twig
+++ b/templates/admin/crud/field/id_with_icon.html.twig
@@ -1 +1,2 @@
-<i class="fa fa-key"></i> {{ field.formattedValue }}
+<i class="fa fa-key"></i>
+{{ include('@EasyAdmin/crud/field/id.html.twig') }}
