diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index cc42784..e896f7f 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -37,8 +37,10 @@ class UserCrudController extends AbstractCrudController
         yield DateField::new('createdAt')
             ->hideOnForm();
         yield TextField::new('specialRole')
-            ->setTemplatePath('admin/crud/field/text_special_role.html.twig');
+            ->setTemplatePath('admin/crud/field/text_special_role.html.twig')
+            ->hideOnForm();
         yield ArrayField::new('roles')
-            ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER');
+            ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER')
+            ->hideOnIndex();
     }
 }
