diff --git a/src/Entity/User.php b/src/Entity/User.php
index c4f397a..39b8b68 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -122,6 +122,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
         return $this;
     }
 
+    public function hasRole(string $role): bool
+    {
+        return \in_array($role, $this->roles, true);
+    }
+
     /**
      * @see PasswordAuthenticatedUserInterface
      */
diff --git a/templates/admin/crud/field/text_special_role.html.twig b/templates/admin/crud/field/text_special_role.html.twig
new file mode 100644
index 0000000..0e48be9
--- /dev/null
+++ b/templates/admin/crud/field/text_special_role.html.twig
@@ -0,0 +1,5 @@
+{# @var ea \EasyCorp\Bundle\EasyAdminBundle\Context\AdminContext #}
+{# @var field \EasyCorp\Bundle\EasyAdminBundle\Dto\FieldDto #}
+{# @var entity \EasyCorp\Bundle\EasyAdminBundle\Dto\EntityDto #}
+{# @var user \App\Entity\User #}
+{% set user = entity.instance %}
