diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 9c828673..2ca1ef76 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -46,6 +46,8 @@ class UserCrudController extends AbstractCrudController
             ->renderAsSwitch(false);
         yield DateField::new('createdAt')
             ->hideOnForm();
+        yield TextField::new('specialRole')
+            ->setTemplatePath('admin/crud/field/text_special_role.html.twig');
         yield ArrayField::new('roles')
             ->setHelp('Available roles: ROLE_SUPER_ADMIN, ROLE_ADMIN, ROLE_MODERATOR, ROLE_USER');
     }
