diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 0ccb11a..2cb3de0 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -47,13 +47,14 @@ class UserCrudController extends AbstractCrudController
         yield DateField::new('createdAt')
             ->hideOnForm();
         yield TextField::new('specialRole')
-            ->setTemplatePath('admin/crud/field/text_special_role.html.twig');
+            ->setTemplatePath('admin/crud/field/text_special_role.html.twig')
+            ->hideOnForm();
 
         $roles = ['ROLE_SUPER_ADMIN', 'ROLE_ADMIN', 'ROLE_MODERATOR', 'ROLE_USER'];
         yield ChoiceField::new('roles')
             ->setChoices(array_combine($roles, $roles))
             ->allowMultipleChoices()
             ->renderExpanded()
-            ->renderAsBadges();
+            ->hideOnIndex();
     }
 }
