diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 4fe56364..d2bc6624 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,8 @@ class UserCrudController extends AbstractCrudController
 
     public function configureCrud(Crud $crud): Crud
     {
-        return parent::configureCrud($crud);
+        return parent::configureCrud($crud)
+            ->setEntityPermission('ADMIN_USER_EDIT');
     }
 
     public function configureFields(string $pageName): iterable
