diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index e8af014..f8c2dd5 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -6,6 +6,7 @@ use App\Entity\Question;
 use App\Field\VotesField;
 use Doctrine\ORM\QueryBuilder;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
 use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
 use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
@@ -30,6 +31,11 @@ class QuestionCrudController extends AbstractCrudController
             ]);
     }
 
+    public function configureActions(Actions $actions): Actions
+    {
+        return parent::configureActions($actions);
+    }
+
     public function configureFields(string $pageName): iterable
     {
         yield IdField::new('id')
