diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index ab740fa..71b2298 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -25,7 +25,8 @@ class UserCrudController extends AbstractCrudController
 
     public function configureCrud(Crud $crud): Crud
     {
-        return parent::configureCrud($crud);
+        return parent::configureCrud($crud)
+            ->setEntityPermission('ADMIN_USER_EDIT');
     }
 
     public function configureFields(string $pageName): iterable
