diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index eade251..d533279 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -68,7 +68,8 @@ class QuestionCrudController extends AbstractCrudController
             ])
             ->setHelp('Preview:');
         yield Field::new('votes', 'Total Votes')
-            ->setTextAlign('right');
+            ->setTextAlign('right')
+            ->setPermission('ROLE_SUPER_ADMIN');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
