diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index be0a492f..cf74b3cb 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -69,7 +69,8 @@ class QuestionCrudController extends AbstractCrudController
             ])
             ->setHelp('Preview:');
         yield VotesField::new('votes', 'Total Votes')
-            ->setTextAlign('right');
+            ->setTextAlign('right')
+            ->setPermission('ROLE_SUPER_ADMIN');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(function ($value, ?Question $question): ?string {
