diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5b6670d..ac32cac 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -68,7 +68,8 @@ class QuestionCrudController extends AbstractCrudController
             ])
             ->setHelp('Preview:');
         yield VotesField::new('votes', 'Total Votes')
-            ->setTextAlign('right');
+            ->setTextAlign('right')
+            ->setPermission('ROLE_SUPER_ADMIN');
         yield AssociationField::new('askedBy')
             ->autocomplete()
             ->formatValue(static function ($value, Question $question): ?string {
