diff --git a/templates/admin/crud/approve_action.html.twig b/templates/admin/crud/approve_action.html.twig
index a19559b2..ef5104b8 100644
--- a/templates/admin/crud/approve_action.html.twig
+++ b/templates/admin/crud/approve_action.html.twig
@@ -1,5 +1,5 @@
 {% if action.name == 'approve' and ea.entity.fqcn == 'App\\Entity\\Question' %}
-    <form action="{{ action.linkUrl }}" method="post">
+    <form action="{{ action.linkUrl }}" method="post" style="margin-right: 10px">
         {{ include('@EasyAdmin/crud/action.html.twig') }}
     </form>
 {% else %}
