diff --git a/templates/admin/approve_action.html.twig b/templates/admin/approve_action.html.twig
index f3407292..dff95a85 100644
--- a/templates/admin/approve_action.html.twig
+++ b/templates/admin/approve_action.html.twig
@@ -1,6 +1,6 @@
 {# @var ea \EasyCorp\Bundle\EasyAdminBundle\Context\AdminContext #}
 {# @var action \EasyCorp\Bundle\EasyAdminBundle\Dto\ActionDto #}
 {# @var entity \EasyCorp\Bundle\EasyAdminBundle\Dto\EntityDto #}
-<form action="{{ action.linkUrl }}" method="POST">
+<form action="{{ action.linkUrl }}" method="POST" class="me-2">
     {{ include('@EasyAdmin/crud/action.html.twig') }}
 </form>
