diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 08c2881..5d2b065 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -147,10 +147,12 @@ class QuestionCrudController extends AbstractCrudController
         parent::deleteEntity($entityManager, $entityInstance);
     }
 
-    public function approve(AdminContext $adminContext)
+    public function approve(AdminContext $adminContext, EntityManagerInterface $entityManager)
     {
         /** @var Question $question */
         $question = $adminContext->getEntity()->getInstance();
         $question->setIsApproved(true);
+
+        $entityManager->flush();
     }
 }
