diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index f89a2af..371e1eb 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -45,6 +45,7 @@ class QuestionCrudController extends AbstractCrudController
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-eye')
             ->setLabel('View');
+        $approveAction = Action::new('approve');
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
@@ -52,7 +53,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission(Action::EDIT, 'ROLE_MODERATOR')
             ->setPermission(Action::NEW, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::DELETE, 'ROLE_SUPER_ADMIN')
-            ->add(Crud::PAGE_DETAIL, $viewAction);
+            ->add(Crud::PAGE_DETAIL, $viewAction)
+            ->add(Crud::PAGE_DETAIL, $approveAction);
     }
 
     public function configureFields(string $pageName): iterable
