diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 0593eda..1444175 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -51,7 +51,10 @@ class QuestionCrudController extends AbstractCrudController
             ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
-            ->displayAsButton();
+            ->displayAsButton()
+            ->displayIf(static function (Question $question): bool {
+                return !$question->getIsApproved();
+            });
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
