diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index b64d35f..2b5e0d8 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -147,4 +147,8 @@ class QuestionCrudController extends AbstractCrudController
 
         parent::deleteEntity($entityManager, $entityInstance);
     }
+
+    public function approve()
+    {
+    }
 }
