diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index c68c9a3..d88a787 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -154,12 +154,14 @@ class QuestionCrudController extends AbstractCrudController
         parent::deleteEntity($entityManager, $entityInstance);
     }
 
-    public function approve(AdminContext $adminContext)
+    public function approve(AdminContext $adminContext, EntityManagerInterface $entityManager)
     {
         $question = $adminContext->getEntity()->getInstance();
         if (!$question instanceof Question) {
             throw new \LogicException('Entity is missing or not a Question');
         }
         $question->setIsApproved(true);
+
+        $entityManager->flush();
     }
 }
