diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 79e576e..e8fde58 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -46,6 +46,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setIcon('fa fa-eye')
             ->setLabel('View');
         $approveAction = Action::new('approve')
+            ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
             ->displayAsButton();
