diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index f81b99f9..6c3d1396 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -46,6 +46,7 @@ class QuestionCrudController extends AbstractCrudController
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-eye')
             ->setLabel('View on site');
+        $approveAction = Action::new('approve');
 
         return parent::configureActions($actions)
             ->update(Crud::PAGE_INDEX, Action::DELETE, static function(Action $action) {
@@ -64,7 +65,8 @@ class QuestionCrudController extends AbstractCrudController
             ->setPermission(Action::DELETE, 'ROLE_SUPER_ADMIN')
             ->setPermission(Action::BATCH_DELETE, 'ROLE_SUPER_ADMIN')
             ->add(Crud::PAGE_DETAIL, $viewAction)
-            ->add(Crud::PAGE_INDEX, $viewAction);
+            ->add(Crud::PAGE_INDEX, $viewAction)
+            ->add(Crud::PAGE_DETAIL, $approveAction);
     }
 
     public function configureFields(string $pageName): iterable
