diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 8772c69..ffd7963 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -47,6 +47,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setIcon('fa fa-eye')
             ->setLabel('View on site');
         $approveAction = Action::new('approve')
+            ->setTemplatePath('admin/approve_action.html.twig')
             ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
diff --git a/templates/admin/approve_action.html.twig b/templates/admin/approve_action.html.twig
new file mode 100644
index 0000000..1095396
--- /dev/null
+++ b/templates/admin/approve_action.html.twig
@@ -0,0 +1,4 @@
+{# @var ea \EasyCorp\Bundle\EasyAdminBundle\Context\AdminContext #}
+{# @var action \EasyCorp\Bundle\EasyAdminBundle\Dto\ActionDto #}
+{# @var entity \EasyCorp\Bundle\EasyAdminBundle\Dto\EntityDto #}
+{{ dump(action) }}
