diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index ffd7963..9e3a926 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -158,4 +158,8 @@ class QuestionCrudController extends AbstractCrudController
 
         parent::deleteEntity($entityManager, $entityInstance);
     }
+
+    public function approve()
+    {
+    }
 }
