diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index e8fde58..d0075a5 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -46,6 +46,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setIcon('fa fa-eye')
             ->setLabel('View');
         $approveAction = Action::new('approve')
+            ->setTemplatePath('admin/crud/approve_action.html.twig')
             ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
diff --git a/templates/admin/crud/approve_action.html.twig b/templates/admin/crud/approve_action.html.twig
new file mode 100644
index 0000000..d73054d
--- /dev/null
+++ b/templates/admin/crud/approve_action.html.twig
@@ -0,0 +1 @@
+{{ dump(action) }}
