diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d0075a5..a1436b8 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -148,4 +148,8 @@ class QuestionCrudController extends AbstractCrudController
 
         parent::deleteEntity($entityManager, $entityInstance);
     }
+
+    public function approve()
+    {
+    }
 }
