diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5e84026..19672b2 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -45,6 +45,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setIcon('fa fa-eye')
             ->setLabel('View');
         $approveAction = Action::new('approve')
+            ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
             ->displayAsButton();
