diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 262d110..8772c69 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -47,6 +47,7 @@ class QuestionCrudController extends AbstractCrudController
             ->setIcon('fa fa-eye')
             ->setLabel('View on site');
         $approveAction = Action::new('approve')
+            ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
             ->displayAsButton();
