diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index d089f75..5c76e63 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -51,7 +51,10 @@ class QuestionCrudController extends AbstractCrudController
             ->linkToCrudAction('approve')
             ->addCssClass('btn btn-success')
             ->setIcon('fa fa-check-circle')
-            ->displayAsButton();
+            ->displayAsButton()
+            ->displayIf(static function (Question $question): bool {
+                return !$question->getIsApproved();
+            });
 
         return parent::configureActions($actions)
             ->setPermission(Action::INDEX, 'ROLE_MODERATOR')
