diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index e90b56a..98d0bc5 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -17,6 +17,7 @@ use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\Field;
 use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\TextareaField;
+use EasyCorp\Bundle\EasyAdminBundle\Router\AdminUrlGenerator;
 
 class QuestionCrudController extends AbstractCrudController
 {
@@ -154,7 +155,7 @@ class QuestionCrudController extends AbstractCrudController
         parent::deleteEntity($entityManager, $entityInstance);
     }
 
-    public function approve(AdminContext $adminContext, EntityManagerInterface $entityManager)
+    public function approve(AdminContext $adminContext, EntityManagerInterface $entityManager, AdminUrlGenerator $adminUrlGenerator)
     {
         $question = $adminContext->getEntity()->getInstance();
         if (!$question instanceof Question) {
@@ -163,5 +164,13 @@ class QuestionCrudController extends AbstractCrudController
         $question->setIsApproved(true);
 
         $entityManager->flush();
+
+        $targetUrl = $adminUrlGenerator
+            ->setController(self::class)
+            ->setAction(Crud::PAGE_DETAIL)
+            ->setEntityId($question->getId())
+            ->generateUrl();
+
+        return $this->redirect($targetUrl);
     }
 }
