diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 0782963..7f2e5c1 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -19,7 +19,8 @@ class QuestionCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->onlyOnIndex();
         yield Field::new('name');
-        yield Field::new('votes', 'Total Votes');
+        yield Field::new('votes', 'Total Votes')
+            ->setTextAlign('right');
         yield Field::new('createdAt')
             ->hideOnForm();
     }
