diff --git a/src/Controller/Admin/TopicCrudController.php b/src/Controller/Admin/TopicCrudController.php
index 27c408e..eb96725 100644
--- a/src/Controller/Admin/TopicCrudController.php
+++ b/src/Controller/Admin/TopicCrudController.php
@@ -7,6 +7,7 @@ use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
 use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\Field;
 use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
 
@@ -34,5 +35,6 @@ class TopicCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->hideOnForm();
         yield Field::new('name');
+        yield AssociationField::new('questions');
     }
 }
