diff --git a/src/Controller/Admin/TopicCrudController.php b/src/Controller/Admin/TopicCrudController.php
index c163e197..13af7560 100644
--- a/src/Controller/Admin/TopicCrudController.php
+++ b/src/Controller/Admin/TopicCrudController.php
@@ -36,6 +36,7 @@ class TopicCrudController extends AbstractCrudController
             ->hideOnForm();
         yield Field::new('name');
         yield AssociationField::new('questions')
-            ->autocomplete();
+            ->autocomplete()
+            ->setCrudController(QuestionCrudController::class);
     }
 }
