diff --git a/src/Controller/Admin/TopicCrudController.php b/src/Controller/Admin/TopicCrudController.php
index eb967250..c163e197 100644
--- a/src/Controller/Admin/TopicCrudController.php
+++ b/src/Controller/Admin/TopicCrudController.php
@@ -35,6 +35,7 @@ class TopicCrudController extends AbstractCrudController
         yield IdField::new('id')
             ->hideOnForm();
         yield Field::new('name');
-        yield AssociationField::new('questions');
+        yield AssociationField::new('questions')
+            ->autocomplete();
     }
 }
