diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 3069d3c..f0694f0 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -22,7 +22,7 @@ class UserCrudController extends AbstractCrudController
     public function configureFields(string $pageName): iterable
     {
         yield IdField::new('id');
-        yield TextField::new('email');
+        yield EmailField::new('email');
         yield TextField::new('fullName');
         yield BooleanField::new('enabled');
         yield DateField::new('createdAt');
