diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index ea748a7e..1fe0bb31 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -21,8 +21,7 @@ class UserCrudController extends AbstractCrudController
     {
         yield IdField::new('id');
         yield TextField::new('email');
-        yield TextField::new('firstName');
-        yield TextField::new('lastName');
+        yield TextField::new('fullName');
         yield BooleanField::new('enabled');
         yield DateField::new('createdAt');
     }
