diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index f533c310..a0b28714 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -50,7 +50,8 @@ class DashboardController extends AbstractDashboardController
      */
     public function configureUserMenu(UserInterface $user): UserMenu
     {
-        return parent::configureUserMenu($user);
+        return parent::configureUserMenu($user)
+            ->setAvatarUrl($user->getAvatarUrl());
     }
 
     public function configureActions(): Actions
