diff --git a/src/Controller/Admin/DashboardController.php b/src/Controller/Admin/DashboardController.php
index 10b877b6..f12c58f4 100644
--- a/src/Controller/Admin/DashboardController.php
+++ b/src/Controller/Admin/DashboardController.php
@@ -48,7 +48,8 @@ class DashboardController extends AbstractDashboardController
      */
     public function configureUserMenu(UserInterface $user): UserMenu
     {
-        return parent::configureUserMenu($user);
+        return parent::configureUserMenu($user)
+            ->setAvatarUrl($user->getAvatarUrl());
     }
 
     public function configureActions(): Actions
