diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index 8777fcbc..9371eda4 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -23,7 +23,8 @@ class UserCrudController extends AbstractCrudController
         yield IdField::new('id');
         yield EmailField::new('email');
         yield TextField::new('fullName');
-        yield BooleanField::new('enabled');
+        yield BooleanField::new('enabled')
+            ->renderAsSwitch(false);
         yield DateField::new('createdAt');
     }
 }
