diff --git a/src/Controller/Admin/UserCrudController.php b/src/Controller/Admin/UserCrudController.php
index f0694f0..eebc566 100644
--- a/src/Controller/Admin/UserCrudController.php
+++ b/src/Controller/Admin/UserCrudController.php
@@ -24,7 +24,8 @@ class UserCrudController extends AbstractCrudController
         yield IdField::new('id');
         yield EmailField::new('email');
         yield TextField::new('fullName');
-        yield BooleanField::new('enabled');
+        yield BooleanField::new('enabled')
+            ->renderAsSwitch(false);
         yield DateField::new('createdAt');
     }
 }
