diff --git a/assets/controllers/editor_controller.js b/assets/controllers/editor_controller.js
new file mode 100644
index 0000000..9a72242
--- /dev/null
+++ b/assets/controllers/editor_controller.js
@@ -0,0 +1,25 @@
+import { Controller } from 'stimulus';
+import snarkdown from 'snarkdown';
+const document = window.document;
+
+export default class extends Controller {
+    static targets = [ 'container', 'input', 'output' ]
+
+    initialize() {
+        const div = document.createElement('div');
+        div.className = 'markdown-preview';
+        div.dataset.editorTarget = 'output';
+        div.textContent = 'MARKDOWN WILL BE RENDERED HERE';
+
+        this.containerTarget.append(div);
+    }
+
+    connect() {
+        this.render();
+    }
+
+    render() {
+        const markdownContent = this.inputTarget.value;
+        this.outputTarget.innerHTML = snarkdown(markdownContent);
+    }
+}
