diff --git a/assets/controllers/snarkdown_controller.js b/assets/controllers/snarkdown_controller.js
new file mode 100644
index 00000000..f4741a01
--- /dev/null
+++ b/assets/controllers/snarkdown_controller.js
@@ -0,0 +1,26 @@
+import { Controller } from '@hotwired/stimulus';
+import snarkdown from 'snarkdown';
+const document = window.document;
+
+export default class extends Controller {
+    static targets = ['input'];
+
+    outputElement = null;
+
+    initialize() {
+        this.outputElement = document.createElement('div');
+        this.outputElement.className = 'markdown-preview';
+        this.outputElement.textContent = 'MARKDOWN WILL BE RENDERED HERE';
+
+        this.element.append(this.outputElement);
+    }
+
+    connect() {
+        this.render();
+    }
+
+    render() {
+        const markdownContent = this.inputTarget.value;
+        this.outputElement.innerHTML = snarkdown(markdownContent);
+    }
+}
