diff --git a/assets/controllers/snarkdown_controller.js b/assets/controllers/snarkdown_controller.js
index f4741a01..d375f69d 100644
--- a/assets/controllers/snarkdown_controller.js
+++ b/assets/controllers/snarkdown_controller.js
@@ -2,6 +2,7 @@ import { Controller } from '@hotwired/stimulus';
 import snarkdown from 'snarkdown';
 const document = window.document;
 
+/* stimulusFetch: 'lazy' */
 export default class extends Controller {
     static targets = ['input'];
 
