diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index 5eea58d..f8a2d0c 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -11,6 +11,7 @@ use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\Field;
 use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
 use EasyCorp\Bundle\EasyAdminBundle\Field\TextareaField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextEditorField;
 
 class QuestionCrudController extends AbstractCrudController
 {
@@ -41,7 +42,7 @@ class QuestionCrudController extends AbstractCrudController
             );
         yield Field::new('name')
             ->setSortable(false);
-        yield TextareaField::new('question')
+        yield TextEditorField::new('question')
             ->hideOnIndex();
         yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right');
