diff --git a/src/Controller/Admin/QuestionCrudController.php b/src/Controller/Admin/QuestionCrudController.php
index a13750a..8f873aa 100644
--- a/src/Controller/Admin/QuestionCrudController.php
+++ b/src/Controller/Admin/QuestionCrudController.php
@@ -43,7 +43,17 @@ class QuestionCrudController extends AbstractCrudController
         yield Field::new('name')
             ->setSortable(false);
         yield TextareaField::new('question')
-            ->hideOnIndex();
+            ->hideOnIndex()
+            ->setFormTypeOptions([
+                'row_attr' => [
+                    'data-controller' => 'editor',
+                    'data-editor-target' => 'container',
+                ],
+                'attr' => [
+                    'data-editor-target' => 'input',
+                    'data-action' => 'input->editor#render',
+                ],
+            ]);
         yield VotesField::new('votes', 'Total Votes')
             ->setTextAlign('right');
         yield AssociationField::new('askedBy')
